demos/bluetooth/mesh/thingy52/common/model/thingy52.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file
+ *  @defgroup bt_mesh_thingy52 Thingy52 models
+ *  @{
+ *  @brief API for the Thingy52 models.
+ */
+
+#ifndef BT_MESH_THINGY52_H__
+#define BT_MESH_THINGY52_H__
+
+#include <bluetooth/mesh/model_types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** RGB colors */
+struct bt_mesh_thingy52_rgb {
+	/** Color Red */
+	uint8_t red;
+	/** Color Green */
+	uint8_t green;
+	/** Color Blue */
+	uint8_t blue;
+};
+
+/** Thingy52 RGB message */
+struct bt_mesh_thingy52_rgb_msg {
+	/** Remaining hops of the message */
+	uint16_t ttl;
+	/** Delay assosiated with the message */
+	uint16_t delay;
+	/** Color of the message. */
+	struct bt_mesh_thingy52_rgb color;
+	/** Speaker status of the message. */
+	bool speaker_on;
+};
+
+/** @cond INTERNAL_HIDDEN */
+
+#define BT_MESH_NORDIC_SEMI_COMPANY_ID 0x0059
+#define BT_MESH_MODEL_ID_THINGY52_SRV 0x0005
```

Comment:
Model IDs `0x0005` and `0x0006` are already reserved by `RSSI Server` and `RSSI Client` models in nRF5 SDK for Mesh: https://infocenter.nordicsemi.com/topic/com.nordic.infocenter.meshsdk.v5.0.0/group__RSSI__COMMON.html?cp=7_2_4_3_7_1_4#ga30297dc76bfe4246f4cf1a0941ed596d

Could you use another model IDs? `0x0002` - `0x0004` and all after 0x0007` are free.

Reply:
I've assigned IDs `0x0008` and 0x0009` for `Thingy52 Server` and `Thingy52 Client` models accordingly in this PR: https://github.com/nrfconnect/sdk-nrf/pull/3549 You can suggest other IDs if you want.